### DISCUSSION TITLE
Missing config.h for Adventure game

### DISCUSSION THREAD

**Discussion by James-HoneyBadger** [picocomputer/community#58] [Q&A] [answered]
Howdy all,

I tried to compile the adventure game & it errored out relating to the config.h include file (last line of advent.h).  I can't find config.h in the repository.  Am I missing something that ought to be blatantly obvious?  Any ideas?

**Comment by rumbledethumps** [comment, marked as answer]
it's there now.

**Reply by James-HoneyBadger** [reply]
thank you, sir!

**Comment by James-HoneyBadger** [comment]
by the way (in case this helps anyone else) .... you'll need to copy the four advent txt files to the pico computer as well

and yup, certainly looks to be the full fledged Colossal Cave game as I recall from back in the day :)
